test/jdk/java/net/HttpURLConnection/SetAuthenticator/HTTPTestClient.java

```diff
@@ -40,6 +41,28 @@ public static void connect(HttpProtocolType protocol,
                                HttpAuthType authType,
                                Authenticator auth)
             throws IOException {
+        try {
+            doConnect(protocol, server, authType, auth);
+        } catch (BindException ex) {
+            // sleep a bit then try again once
+            System.out.println("WARNING: Unexpected BindException: " + ex);
+            System.out.println("\tSleeping a bit and try again...");
+            System.gc();
+            try {
+                Thread.sleep(500);
+            } catch (InterruptedException iex) {
```

Comment:
If it only fails rarely, would it be safer to delay a few seconds to allow more time for resources to be freed?

Reply:
Good point. What delay would you recommend?

Reply:
2-3 sec maybe